contracts/LendingPoolCore.vy

```diff
@@ -240,12 +240,13 @@ def setLendingPoolPeripheralAddress(_address: address):
 
 
 @external
-def deposit(_lender: address, _amount: uint256) -> bool:
+def deposit(_lender: address, _payer: address, _amount: uint256) -> bool:
```

Comment:
In which scenarios is the `_payer` used?

Reply:
It's needed to allow both deposits in ETH (the `LendingPoolPeripheral` is wrapping and paying) and WETH (the lender is paying)